packages/ckeditor5-image/tests/imageinsert/imageinsertviaurlui.js

```diff
@@ -186,6 +191,10 @@ describe( 'ImageInsertViaUrlUI', () => {
 
 		testSubmit( 'accept button', () => acceptButton.fire( 'execute' ) );
 
+		/**
+	 	* Browsers handle clicking Enter for form submitting by its own but unit test can not, it is known limitation.
+		* We fire a form submit event to simulate the behavior that occurs when pressing Enter in the browser.
+	 	*/
```

Comment:
```suggestion
		/**
	 	* Browsers handle pressing Enter on forms natively by submitting it. We fire a form submit event to simulate that behavior.
	 	*/
```

Reply:
I think it could be a bit shorter as above. And a "nitpick", `enter` is pressed not clicked (usually at least 😄).